cpp/harvest/marc_issn_lookup.cc

```diff
@@ -0,0 +1,171 @@
+/**
+ * \brief Utility for updating issn information
+ * \author Steven Lolong (steven.lolong@uni-tuebingen.de)
+ *
+ * \copyright 2023 Tübingen University Library.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <fstream>
+#include <iostream>
+#include "CORE.h"
+#include "FileUtil.h"
+#include "MARC.h"
+#include "util.h"
+
+namespace {
+[[noreturn]] void Usage() {
+    ::Usage(
+        "marc_input_articles marc_input_journals marc_output_articles"
+        "\n"
+        "- marc_input_artiles is a file containing all article information taken from CORE. \n"
+        "- marc_input_journals is a file containing journal information. Please use issn_lookup.py to generate this file. \n"
+        "- marc_output_articles is an output file generated by this tool. \n");
+
+    std::exit(EXIT_FAILURE);
+}
+struct SubFieldInfo {
+    std::string i;
```

Comment:
According to our internal coding convention, all class/struct member variable names must end with an underscore, e.g. `i` => `i_` and so on.